packages/backend/src/build-disk-image.ts

```diff
@@ -100,12 +100,13 @@ export async function buildDiskImage(build: BootcBuildInfo, history: History): P
   // "Returning" withProgress allows PD to handle the task in the background with building.
   return extensionApi.window
     .withProgress(
-      { location: extensionApi.ProgressLocation.TASK_WIDGET, title: 'Building disk image ' + build.name },
+      { location: extensionApi.ProgressLocation.TASK_WIDGET, title: 'Building disk image ' + build.id },
```

Comment:
I think we should keep this one to build.image (or build.image+":"+build.tag). If I launch it several days in a row I'd expect the tasks to still say it's building "image-name", not "image-name 5".